src/components/mediaViewer/README.md

```diff
@@ -0,0 +1,16 @@
+## **Media viewer**
+
+### Description:
+
+Media viewer with possibility of opening images/video in new modal.
+Thumbnails or other resources should be in 'static/img' folder.
```

Comment:
Can we use images from their md folders?
For example docs/dashboards-and-widgets/img/widget-types instead static/img/widget-types

Reply:
No, we should move all images to static/img folder